lib/src/main/res/values/attrs.xml

```diff
@@ -1,14 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <declare-styleable name="TextElement">
-        <attr name="textColor" format="reference|color" />
-        <attr name="text" format="string" />
         <attr name="hint" format="string" />
-        <attr name="removeDefaultStyles" format="boolean" />
-        <attr name="mask" format="string" />
         <attr name="keyboardType" format="enum">
             <enum name="text" value="1" />
             <enum name="number" value="2" />
         </attr>
+        <attr name="mask" format="string" />
```

Comment:
is there any reason you reordered this?

Reply:
I alphabetized the list, it was getting unruly.